logging/Limiting_Access_to_Logs.md

```diff
@@ -122,23 +204,52 @@ and roles and between users and back-end roles.
+| production_acme_kibana_users | search_index_production_acme | Grants access to log messages from the `production` namespace and `acme` tenant |
 
-After these access controls have been defined, you can assign the back-end role of `production_acme_kibana_users` to a user. The back-end role 
+After these access controls have been defined, you can assign the back-end role 
+of `production_acme_kibana_users` to a user. The back-end role 
 enables the user to access only:
--  log messages collected from the pods associated with the SAS Viya tenant `acme` in the `production` namespace
-- Kibana `production_acme` Kibana tenant space  
-
-In some cases, you might want to create users who cannot log into Kibana but can access the log messages collected from a specific namespace or namespace_tenant combination. These users might be needed to allow some automated process to extract log messages through API calls. To do this, assign the `search_index_ production_acme` role to the user, rather than the `production_acme_kibana_users` back-end role. Because the user has not 
+-  log messages collected from the pods associated with the SAS Viya tenant 
+  `acme` in the `production` namespace
+- Kibana-tenant space `production_acme` 
+
+In some cases, you might want to create users who cannot log in to Kibana but 
+can access the log messages collected from a specific namespace or 
+namespace_tenant combination. These users might be needed to allow some 
+automated process to extract log messages through API calls. To do this, 
+assign the `search_index_production_acme` role to the user, rather than 
+the `production_acme_kibana_users` back-end role. Because the user has not 
 also been assigned the `v4m_kibana_user` or `tenant_production_acme` 
-roles, the user cannot sign into Kibana and cannot access the 
-`production_acme` Kibana tenant space. 
+roles, the user cannot log in to Kibana and cannot access the 
+`production_acme` Kibana-tenant space.
+
+### Roles and Role Mappings for the `logadm` User
+
+As of the 1.1.3 release, the following items are created during the deployment 
+process:
 
-You can also use these roles and back-end roles to create users that can access multiple tenants.  
+- a set of role-based access controls (RBACs)
+- the `logadm` user, which is linked to the new RBACs
+  
+| Back-end Role | Role | Purpose |
+| --- | --- | --- |
+|     | v4m_kibana_user | Grants user access to the Kibana functionality needed. (This role is not unique to the `logadm` user). |
+|     | search_index | Grants access to all collected log messages. |
+|     | tenant_cluster_admins | Grants access to the cluster_admins Kibana-tenant space.|
+|V4MCLUSTER_ADMIN_kibana_users |    | Provides an easy way to link the above roles with a user.
 
+Linking users to the back-end role V4MCLUSTER_ADMIN_kibana_users via the 
+Kibana security plug-in grants those users the access described in the table above.
```

Comment:
Maybe add the example of using the user.sh script with the special values of "\_all\_" for namespace and tenant here?

Reply:
Content added to this section:
You can also use the user.sh script to grant a user the same level of access as 
a `logadm` user. See the following syntax for an example:

<pre>
logging/bin/user.sh CREATE --user <i>user_name</i> --password <i>password</i> 
--namespace <i>namespace</i> --tenant <i>tenant</i>
</pre>

Where: 

- *user_name* is "chen".
- *password* is the password to sign in to Kibana as the user "chen".
- *namespace* is `_all_`.
- *tenant* is `_all_`.